toolkit/tools/srpmpacker/srpmpacker.go

```diff
@@ -96,6 +98,8 @@ type specState struct {
 	err      error
 }
 
+var errPackerCancelReceived = fmt.Errorf("packer cancel signal received")
```

Comment:
Can we move this inside `hydrateFromRemoteSource`? It seems that's the only place we use it.